iocore/cache/Cache.cc

```diff
@@ -590,12 +590,25 @@ CacheProcessor::start_internal(int flags)
   check                = (flags & PROCESSOR_CHECK) != 0;
   start_done           = 0;
   Span *sd;
+  char **paths;
+  int *fds;
+  int *sector_sizes;
```

Comment:
I suggest we move the sd declaration down to it's use in the for loop, and move the rest of these declarations to their initialization a few lines below.

Reply:
Sure, I'll tidy that up.